recipes-aos/aos-iamanager/aos-iamanager_git.bb

```diff
@@ -56,7 +56,11 @@ S = "${WORKDIR}/git"
 do_compile[network] = "1"
 do_configure[network] =  "1"
 
-do_fetch[vardeps] += "AOS_MAIN_NODE AOS_MAIN_NODE_HOSTNAME AOS_NODE_HOSTNAME AOS_NODE_TYPE AOS_RUNNER"
+do_fetch[vardeps] += " \
+    AOS_MAIN_NODE AOS_MAIN_NODE_HOSTNAME AOS_NODE_HOSTNAME AOS_NODE_TYPE \
+    AOS_RUNNER AOS_CPU_MODEL_NAME AOS_CPU_ARCHITECTURE AOS_CPU_VARIANT \
```

Comment:
We should remove AOS_RUNNER. I don't think we need to set AOS_CPU_MODEL_NAME with yocto.
The following vars should be populated by yocto:
AOS_ARCH
AOS_ARCH_VARIANT
AOS_OS
AOS_OS_VERSION